instrumentation/jmx-metrics/testing-webapp/src/main/java/io/opentelemetry/instrumentation/jmx/testapp/JakartaSimpleServlet.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jmx.testapp;
+
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+// servlet that will be used for jakarta servlet containers
+@WebServlet(urlPatterns = "/jakarta/*", name = "jakartaServlet")
+public class JakartaSimpleServlet extends HttpServlet {
+
+  private static final long serialVersionUID = 3726145372238690099L;
```

Comment:
Usually one would set the `serialVersionUID` to either 1 or 0, curious why you chose that particular number.